README.md

```diff
@@ -10,6 +10,8 @@ The following types of tokens are supported:
 
 - Web3 NFT Tokens (On Chain)
```

Comment:
Could we change this title to be, "Non-Fungible tokens"

This will better align this part of the README with the newly added feature, Fungible tokens (ERC20)

e.g.

- Non-Fungible tokens
- Fungible tokens (ERC20)